libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/FeatureList/FeatureList.tsx

```diff
@@ -107,7 +107,14 @@ export class FeatureList extends React.Component<
     this.layerHostId = getId("featuresListHost");
     this._selection = new Selection({
       onSelectionChanged: (): void => {
-        const newSelectedFeatures = this.getSelectionDetails();
+        let newSelectedFeatures = this.getSelectionDetails();
+        const oldSelectedFeaturesNotSearched = this.state.selectedFeatures.filter(
+          (oldSelectedFeature) =>
+            !this.state.searchedFeatures.includes(oldSelectedFeature)
+        );
+        newSelectedFeatures = newSelectedFeatures.concat(
+          oldSelectedFeaturesNotSearched
```

Comment:
why do we need to concat non-searched feature?

Reply:
so the selection is only for the active/searched items in the features list.  If there were items already selected before search, and this logic is removed, suddenly they get unselected when a user changes selection for a searched feature.